itests/org.openhab.binding.modbus.tests/src/main/java/org/openhab/binding/modbus/tests/ModbusPollerThingHandlerTest.java

```diff
@@ -374,24 +374,6 @@ protected boolean matchesSafely(ModbusReadCallback callback) {
         verifyNoMoreInteractions(mockedModbusManager);
     }
 
-    @Test
```

Comment:
This test is no longer working because core now prevents initialization of things without a bridge if the thing requires a bridge.